papers/bcr-0004-bc32.md

```diff
@@ -0,0 +1,51 @@
+# The BC32 Data Encoding Format
+## BCR-0004
+
+**© 2020 Blockchain Commons**
+
+Author: Wolf McNally<br/>
+Date: May 2, 2020
+
+---
+
+### Introduction
+
+The [Bech32] checksummed base32 format was introduced to transport segregated witness signatures for Bitcoin. It has several desirable qualities:
+
+* The payload and checksum data use a limited character set compatible with both URI syntax and QR code alphanumeric encoding mode.
+* The character set uses the letters A-Z in a case-modal fashion (i.e., one can use all upper case or all lower case but the algorithm rejects mixed-case.)
+* The character set is chosen to minimize ambiguity, and the ordering is chosen to minimize the number of pairs of similar characters.
+* Built-in error detection.
+* Because the character set used are all "word characters", most text editors will select the entire sequence of characters when it is double-clicked.
+
+However, it also has a couple drawbacks when considered for general data encoding:
+
+* The checksum algorithm is suited to a smaller number of bytes.
+* The characters allowed for the human readable part ("HRP") conflict with both URI syntax and QR code alphanumeric encoding mode.
+
+### Proposal
+
+This document proposes a new, more general data encoding format *BC32* that is based on Bech32 but makes the following modifications:
+
+* The HRP and numeral `1` divider are no longer included.
+* **TODO:** The checksum is enhanced to handle larger payloads.
```

Comment:
parameters are defined to handle error correction / identification for larger payloads.

Reply:
Fixed.